src/main/java/uk/gov/justice/hmpps/casenotes/types/CaseNoteTypeController.kt

```diff
@@ -129,9 +131,25 @@ class CaseNoteTypeController(
       ],
     ),
   )
-  @Operation(summary = "Retrieves a list of case note types")
+  @Operation(
+    summary = "Retrieves a list of case note types",
+    description =
+    """
+  Please note, current functionality of using user roles from the token is now deprecated. 
+  Going forward user roles will not affect the results returned from this endpoint.
+  This has been replaced with optional request params to replicate the same functionality.
+  Setting a value for 'selectableBy' will allow the choice of filtering those that are selectable by dps users only or to include all types regardless.
+  Additionally, a list of values are available for the include filter: 
+  INACTIVE -> if this is present the returned results will include inactive types, otherwise only active types will be returned.
```

Comment:
Worth adding that not supplying the include parameter will return all active and inactive subtypes